bcda/cli.go

```diff
@@ -157,3 +148,61 @@ func getCCLFFileMetadata(filePath string) (cclfFileMetadata, error) {
 
 	return metadata, nil
 }
+
+func importCCLFDirectory(filePath string) (success, failure int, err error) {
+	var cclf0, cclf8, cclf9 []cclfFileMetadata
+
+	err = filepath.Walk(filePath, sortCCLFFiles(&cclf0, &cclf8, &cclf9))
+	if err != nil {
+		return 0, 0, err
+	}
+	for _, file := range cclf8 {
+		//todo match cclf0 and validate
+		err = importCCLF8(file)
+		if err != nil {
+			failure += 1
```

Comment:
We should probably log the failure with accompanying filename.

Reply:
I think this is a concern for importCCLF8, not this code.